stacks/static/service-config/mime.types

```diff
@@ -0,0 +1,89 @@
+
```

Comment:
That would mean that either:

1. can't let the `service-config/` folder ship symlinks (which I think could be useful)
2. have to write some manifest format for apps to specify 

I mean, while throwing out ideas, we could also ship a symlink to `/etc/nginx/mime.types`, which is likely to have the desired effect, except that then we're relying on the Debian package layout again...